docs/config.md

```diff
@@ -198,9 +198,11 @@ the `ASPNETCORE_HOSTINGSTARTUPASSEMBLIES` environment variable to
 
 ### Instrumentation options
 
-| Environment variable                    | Description                                                                                                                                                        | Default value | Status                                                                                                                            |
-|-----------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------|
-| `OTEL_DOTNET_AUTO_GRAPHQL_SET_DOCUMENT` | Whether GraphQL instrumentation can pass raw queries as `graphql.document` attribute. This may contain sensitive information and therefore is disabled by default. | `false`       | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| Environment variable                                            | Description                                                                                                                                                | Default value | Status                                                                                                                            |
+|-----------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| `OTEL_DOTNET_AUTO_ENTITYFRAMEWORKCORE_SET_DBSTATEMENT_FOR_TEXT` | Whether the Entity Framework Core instrumentation can pass text queries through the `db.statement` attribute. Queries might contain sensitive information. | `false`       | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
```

Comment:
This is not only for "text queries" but for all "SQL statements".  We should tell that if it is `false` (default) then only executions of  stored procedures pass a value to `db.statement`.